### DISCUSSION TITLE
An existing SAN certificate already contains the domain

### DISCUSSION THREAD

**Discussion by dteruel-admin** [rcl-ssl/docs#36] [General]
Hi

I am trying to create a new standalone certificate and I am getting the following error:

ERROR: An existing SAN certificate already contains the domain *.matriz.seicho-no-ie.org.br. Please delete this certificate and try again

But the problem is that I can't delete the cert as someone who used to work for my organisation created the original sert - which has now expired.

How do I move forward and create a new cert?

Thanks

**Comment by rcladmin** [comment]
You can try it now
